build/stage-build-security.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
```

Comment:
I'm not sure why there is `build` in the file name? This stage is not about building the code.

Reply:
renamed to stage-security-scan